package.json

```diff
@@ -40,7 +41,11 @@
     "nan": "2.17.0",
     "node-abi": "^3.34.0",
     "cpu-features": "./.yarn/packages/empty-package",
-    "electron-builder": "23.0.2"
+    "electron-builder": "23.0.2",
```

Comment:
I think these should be in apps/studio/package.json

Reply:
Moving the jest packages to apps/studio doesn't work somehow.